sql/core/src/test/scala/org/apache/spark/sql/FileBasedDataSourceSuite.scala

```diff
@@ -92,4 +96,39 @@ class FileBasedDataSourceSuite extends QueryTest with SharedSQLContext {
       }
     }
   }
+
+  // Only ORC/Parquet support this.
```

Comment:
Wait .. don't other sources support this option?

Reply:
Ur, let me check back.